src/components/Input/__tests__/Input.spec.tsx

```diff
@@ -2,14 +2,24 @@ import React from 'react';
 import { render, RenderResult, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import '@testing-library/jest-dom';
+import { InputProps } from '@textkernel/oneui';
```

Comment:
NIP: I would import it as `import { Props as InputProps } from '../Input'` (so merge with the next line).